components/tests/python/library/__init__.py

```diff
@@ -116,6 +116,18 @@ def teardown_class(cls):
         cls.root = None
         cls.__clients.__del__()
 
+    def keepRootAlive(self):
+        """
+        Keeps root connection alive.
+        """
+        try:
+            if self.root.sf is None:
+                assert self.root.connect()
```

Comment:
Of course this also means this condition was never true and so is just a simple keepalive (the else clause) valid here of is it better to try to create the session again?

`enableKeepAlive` was mentioned in conversation with @jburel but I shied away from it because some of our tests don't clean up that well. Maybe this is something for a card when we do some test refactoring - as I am sure we will do.